### DISCUSSION TITLE
Can you provide an example that supports the Chinese large model?

### DISCUSSION THREAD

**Discussion by tianchiguaixia** [explosion/spacy-llm#384] [Ideas]
Currently, Chinese large models are not supported

**Comment by rmitsch** [comment]
Hey @tianchiguaixia, thanks for suggesting this. We have this on our roadmap.

**Comment by tianchiguaixia** [comment]
Thank you very much, spacy_ Llm has been very helpful to Chinese users. Here is how I use space_ Some demos made by LLM


https://github.com/explosion/spacy-llm/assets/29837553/3f4fb9f7-e267-44f9-8fde-458445850dc0

**Comment by tianchiguaixia** [comment]
https://github.com/explosion/spacy-llm/assets/29837553/bee78e67-e5d6-481b-b57e-3976827d9b9d

**Comment by tianchiguaixia** [comment]
https://github.com/explosion/spacy-llm/assets/29837553/363ad9f9-065e-4125-b6e2-1ede0d9abaa9

**Comment by tianchiguaixia** [comment]
https://github.com/explosion/spacy-llm/assets/29837553/af233269-aa32-4329-91f2-7770c9c9b5d8
